src/main/java/org/sunbird/portal/department/model/ObjectData.java

```diff
@@ -0,0 +1,56 @@
+package org.sunbird.portal.department.model;
+
+import javax.annotation.Generated;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({ "id", "type" })
+
+public class ObjectData {
```

Comment:
Need to move this to another new package which should be separate for telemetry related classes.